package.json

```diff
@@ -464,8 +453,7 @@
           "command": "cmake.resetState"
         },
         {
-          "command": "cmake.configureAll",
-          "when": "cmake:multiRoot && cmake:enableFullFeatureSet"
```

Comment:
Don't remove the `cmake:multiRoot` criteria.

Reply:
good catch